client/rpc/block.go

```diff
@@ -64,7 +64,7 @@ func getBlock(cliCtx context.CLIContext, height *int64) ([]byte, error) {
 
 	// TODO move maarshalling into cmd/rest functions
 	// output, err := tmcodec.MarshalJSON(res)
-	output, err := cdc.MarshalJSON(res)
+	output, err := cdc.MarshalJSONIndent(res, "", "  ")
```

Comment:
Why? Do we necessarily need to send extra bytes back?

Reply:
That is not extra bytes. Just add indent to make the response more elegant.
```
{"type": "cosmos-sdk/Send","value": {"inputs": [{"address": "cosmos1g9ahr6xhht5rmqven628nklxluzyv8z9jqjcmc","coins": [{"denom": "steak","amount": "10"}]}],"outputs": [{"address": "cosmos1x6nfu5eqmmnrycggca626tsj9wecdzpmp5m40q","coins": [{"denom": "steak","amount": "10"}]}]}}
```
MarshalJSONIndent will make the above json string to the following pattern:
```
{
  "type": "cosmos-sdk/Send",
  "value": {
	"inputs": [
	  {
		"address": "cosmos1g9ahr6xhht5rmqven628nklxluzyv8z9jqjcmc",
		"coins": [
		  {
			"denom": "steak",
			"amount": "10"
		  }
		]
	  }
	],
	"outputs": [
	  {
		"address": "cosmos1x6nfu5eqmmnrycggca626tsj9wecdzpmp5m40q",
		"coins": [
		  {
			"denom": "steak",
			"amount": "10"
		  }
		]
	  }
	]
  }
}
```

Reply:
I personally like the indented response more, but I guess it's nbd

Reply:
JSON response is expected to be parsed by machines. Machines usually don't bother about elegance, and users that want pretty outputs can always install dedicated extensions in their browsers

Reply:
But we also provide swagger-ui to access to these APIs. This improvement can make the response on swagger-ui more readable. Besides, the improvement won't in any inconvenience to machines. So how about keep this improvement?

Reply:
So I guess swagger-ui cannot handle unintended JSON, can it?

Reply:
Swagger-ui can handle this well. Indent can prettify the response json, just as my [example]()

> That is not extra bytes. Just add indent to make the response more elegant.
> 
> ```
> {"type": "cosmos-sdk/Send","value": {"inputs": [{"address": "cosmos1g9ahr6xhht5rmqven628nklxluzyv8z9jqjcmc","coins": [{"denom": "steak","amount": "10"}]}],"outputs": [{"address": "cosmos1x6nfu5eqmmnrycggca626tsj9wecdzpmp5m40q","coins": [{"denom": "steak","amount": "10"}]}]}}
> ```
> MarshalJSONIndent will make the above json string to the following pattern:
> 
> ```
> {
>   "type": "cosmos-sdk/Send",
>   "value": {
> 	"inputs": [
> 	  {
> 		"address": "cosmos1g9ahr6xhht5rmqven628nklxluzyv8z9jqjcmc",
> 		"coins": [
> 		  {
> 			"denom": "steak",
> 			"amount": "10"
> 		  }
> 		]
> 	  }
> 	],
> 	"outputs": [
> 	  {
> 		"address": "cosmos1x6nfu5eqmmnrycggca626tsj9wecdzpmp5m40q",
> 		"coins": [
> 		  {
> 			"denom": "steak",
> 			"amount": "10"
> 		  }
> 		]
> 	  }
> 	]
>   }
> }
> ```